### PR TITLE
feat: removing unused vars

### PR DESCRIPTION
**Pull request by danielgent**
Done by setting `"noUnusedLocals": true,` in the tsconfig.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "paths": {
       "*": ["node_modules/*"]
     },
-    "noUnusedLocals": false,
+    "noUnusedLocals": true,
```
Comment (danielgent) [inline tsconfig.json:25]: :eyes: I need to remember not to commit this! :smile:

File: src/routes/oauth2/logout.ts [hunk 2]
```diff
@@ -54,16 +54,17 @@ export const logoutRoutes = new OpenAPIHono<{ Bindings: Env }>()
...
+          // const session = await ctx.env.data.sessions.get(
+          //   client.tenant_id,
+          //   tokenState,
+          // );
+          // if (session) {
+          //   const user = await ctx.env.data.users.get(
+          //     client.tenant_id,
+          //     session.user_id,
+          //   );
+          // }
```
Comment (danielgent) [inline src/routes/oauth2/logout.ts:67]: @markusahlstrand should we be doing something with this user object?

Comment (markusahlstrand) [reply src/routes/oauth2/logout.ts:67]: I guess it was for logging purposes at some point, but it was lost in the refactor

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**